config.json

```diff
@@ -351,6 +364,16 @@
         "prerequisites": [],
         "difficulty": 2
       },
+      {
+        "slug": "acronym",
+        "name": "Acronym",
+        "uuid": "db2ad0c2-293e-11ee-be56-0242ac120002",
```

Comment:
I forgot to tell you it has to be uuid v4, either you can generate a new one or here is otherwise another which should work: `84e522fc-d042-43fe-8dd2-84c9f1150d1e`